library/artifactory_repo.py

```diff
@@ -232,17 +239,9 @@
     artifactory_url: https://artifactory.repo.example.com
     username: your_username
     password: your_pass
-    repo: "test-virtual-update"
-    state: present
-    repo_config: '{"description": "New public description."}'
-
-# Update a virtual repository and register current config after update.
-- name: update test-virtual-update repo
-  artifactory_repo:
-    artifactory_url: https://artifactory.repo.example.com
-    auth_token: your_token
-    repo: "test-virtual-update"
+    name: test-virtual-update
     state: present
+    repo_config_dict:
```

Comment:
still didnt add anything here :P

Reply:
This line actually needs to be removed. The example is two tasks down.